lib/elixir/lib/io/ansi.ex

```diff
@@ -170,6 +170,29 @@ defmodule IO.ANSI do
   @doc "Sends cursor home."
   defsequence(:home, "", "H")
 
+  @doc """
+  Sends cursor to the absolute position specified by `lines`, `columns`, 
+  with 0,0 being the top left corner.
+  """
+  @spec cursor(integer, integer) :: String.t()
+  def cursor(line, column) when line >= 0 and column >= 0, do: "\e[#{line};#{column}H"
```

Comment:
If you want to check for the arguments being also integers, you should explicitly have a `is_integer/1` check. `>` and `>=` works for floats as well.

Please do this on all other functions as well :).